docs/book/src/reference/envtest.md

```diff
@@ -225,7 +225,7 @@ Following an example.
 
         By("installing the cert-manager")
         Expect(utils.InstallCertManager()).To(Succeed())
-    }
+    })
```

Comment:
We will need to get merged first: https://github.com/kubernetes-sigs/kubebuilder/pull/3722
then rebase this one with master to get it merged.